zproject/default_settings.py

```diff
@@ -708,3 +708,7 @@
 INPUT_COST_PER_GIGATOKEN: int = 0
 OUTPUT_COST_PER_GIGATOKEN: int = 0
 MAX_PER_USER_MONTHLY_AI_COST: float | None = 0.5
+
+# URL of the onboarding video displayed to new users.
+# Set it to None to disable it.
+ONBOARDING_VIDEO_URL: str | None = "https://static.zulip.com/static/onboarding-video/zulip-10.mp4"
```

Comment:
`static.zulipchat.com`, not `static.zulip.com`

Reply:
Yeah, will update with https://github.com/zulip/zulip/pull/32074#discussion_r1977942708

This is not currently being used.

Reply:
Done